### PR TITLE
Add Russian language and fix FileChooserAction to open path with Cyri…

### PR DESCRIPTION
**Pull request by camellan**
…llic characters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,12 @@
 
 #### From source
 ```
-git clone https://github.com/BharatKalluri/Gifup
+git clone https://github.com/camellan/Gifup
```
Comment (BharatKalluri) [inline README.md]: can you please revert back to this url

**Comment by BharatKalluri** [comment]
@camellan  Many thanks for the PR, I apologize for the very long delay in the review. I will approve and merge once you make the minor change mentioned above. Thanks!

**Comment by camellan** [comment]
Done

**Comment by BharatKalluri** [comment]
Thanks!

**Pull request merged**